sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala

```diff
@@ -271,6 +271,23 @@ trait TPCDSBase extends SharedSparkSession {
       """.stripMargin
   )
 
+  private val tablePartitionColumns = Map(
```

Comment:
nit: Could you leave some comments about how we decided these partition keys?

Reply:
ok